src/bootstrap/build/flags.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use std::fs;
+use std::path::PathBuf;
+use std::process;
+use std::slice;
+
+use getopts::Options;
+
+pub struct Flags {
+    pub verbose: bool,
+    pub stage: Option<u32>,
+    pub build_target: String,
+    pub host: Filter,
+    pub target: Filter,
+    pub build: Filter,
```

Comment:
Ok, I realize this is probably not referring to the build triple but a list of 'top-level' targets to build. Can you rename this too, and it? What's build_target? Is *that* the 'build' target triple?